src/frontends/pytorch/src/op_table.cpp

```diff
@@ -121,7 +121,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"aten::expand", op::translate_expand},
         {"aten::expand_as", op::translate_expand_as},
         {"aten::flatten", op::translate_flatten},
-        {"aten::floordiv", op::translate_floordiv},
```

Comment:
Why you delete `floordiv`?

Reply:
@mvafin Oh I thought it is the same operator. I have reverted the change and add seperate file for `aten::floor_divide`